release/air_tests/air_benchmarks/workloads/gpu_batch_inference.py

```diff
@@ -113,28 +113,28 @@ def __call__(self, batch: Dict[str, np.ndarray]) -> Dict[str, np.ndarray]:
 
     # For structured output integration with internal tooling
     results = {"data_directory": data_directory, "data_format": data_format}
-    results["perf_metrics"] = [
-        {
+    results["perf_metrics"] = {
+        "total_time_s": {
```

Comment:
We can remove perf_metric_name since it’s already in the key and I don’t think perf_metric_type is used anywhere either.

Reply:
i was thinking we could keep these in case there were other potential future usages for classifying the types of metrics? i saw in a few other benchmarks that were using the same convention